components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -2148,6 +2148,7 @@ spec:
             namespace }}/pipelinerun/{{ pr }}
           custom-console-url-pr-tasklog: https://konflux.apps.stone-stage-p01.hpmt.p1.openshiftapps.com/application-pipeline/ns/{{
             namespace }}/pipelinerun/{{ pr }}/logs/{{ task }}
+          remember-ok-to-test: "false"
```

Comment:
this is not part of the change. maybe a previous change and deploy.yaml not regenerated? Is it consistent with the expected `remember-ok-to-test` value?

Reply:
Yes your right it was generated by the script, i think this PR https://github.com/redhat-appstudio/infra-deployments/pull/4791 was not complete, and it didnt use the generated script
i guess this change is intended
@arewm @mmorhun can you confirm that you want this configuration for remember-ok-to-test

Reply:
this is the value we have in prod, it is ok to have it in stage

Reply:
Let's fix this in a separate PR: https://github.com/redhat-appstudio/infra-deployments/pull/4818

Reply:
PR is merged, can you rebase this PR

Reply:
done